apps/communications/contacts/test/marionette/details_test.js

```diff
@@ -110,4 +110,28 @@ marionette('Contacts > Details', function() {
     var menuOptions = sysMenu.findElements('button');
     assert.ok(menuOptions.length >= 2);
   });
+
+  test('Fav Contact in Reg/Fav List', function() {
```

Comment:
I'd leave a more self-explanatory sentence. Something like: 'Favorite contact should appear in favorite and regular lists'

As we already have another test for favorites, we can use [mocha's TDD APIs](http://mochajs.org/#tdd) more efficiently. For instance:
```js
suite('Favorite contact', function() {
  test('should be marked as favorite in details view', function() {
    // test
  });

  test('should appear in favorite and regular lists', function() {
    // test
  });
})
```